main.tf

```diff
@@ -285,7 +285,7 @@ resource "google_kms_key_ring" "key_ring" {
 resource "google_kms_crypto_key" "crypto_key" {
   name            = local.uuid
   key_ring        = google_kms_key_ring.key_ring.self_link
-  rotation_period = "86400s"
+  rotation_period = "2592000s" # 30 days
```

Comment:
Since afaik the main use of this tf is for our internal testing, and this doesn't solve the problem so much as make it so we're unlikely to hit it in regular usage (ie should we destroy on day 30 we still run into this), I wonder if it'd be prudent to make this value configurable and set it to something ridiculously high? Should this be used on a production deployment, we can set it to something more normal, but otherwise it's set to, say, six months?